trouble_client/src/game/socket_context/index.js

```diff
@@ -2,14 +2,15 @@ import React, { useState, useEffect } from "react";
 import SocketContext from "./context"
 import { initSockets } from "./sockets"; 
 //       ^ initSockets is shown later on
-import SpaceState from "../SpaceState";
+import SpaceState from "../../models/SpaceState";
 import { PlayerColors } from "../../Colors";
+import Player from "../../models/Player";
 
 const SocketProvider = (props) => {
     const [value, setValue] = useState({
       rollResult: 0,
-      currentPlayer: "Player 1",
-      completedPlayer: "Player 2",
+      currentPlayer: new Player("Brad Pitt", PlayerColors.blue),
```

Comment:
so we would compare if currentPlayer.name == my.name of some kind? just trying to figure out how we know when it's "my" turn.

Reply:
Yeah something like that. The other alternative is the server explicitly tells you it's your turn.